pkg/retry/retryer.go

```diff
@@ -318,19 +333,8 @@ func (rc *Container) WithRetryIfNeeded(ctx context.Context, callOptRetry *Policy
 		return ri, false, err
 	}
 
-	// case 3: retry
-	// reqOp is used to ignore req concurrent write
-	reqOp := OpNo
-	// respOp is used to ignore resp concurrent write and read, especially in backup request
-	respOp := OpNo
-	ctx = context.WithValue(ctx, CtxReqOp, &reqOp)
-	ctx = context.WithValue(ctx, CtxRespOp, &respOp)
```

Comment:
`reqOp` and `respOp` are moved up to line 291 `PrepareRetryContext`, and it's invoked in `kClient.Call` (before all possible retry)